arelle/plugin/validate/ESEF/ESEF_Current/DTS.py

```diff
@@ -38,7 +39,7 @@ def checkFilingDTS(val: ValidateXbrl, modelDocument: ModelDocument, esefNotesCon
 
     isExtensionDoc = isExtension(val, modelDocument)
     filenamePattern = filenameRegex = None
-    anchorAbstractExtensionElements = val.authParam["extensionElementsAnchoring"] == "include abstract"
+    anchorAbstractExtensionElements = is2022DisclosureSystem(val.modelXbrl) and val.authParam["extensionElementsAnchoring"] == "include abstract"
```

Comment:
Here and elsewhere, this looks like it should be checking if the disclosure system is older than 2023 (2021 or 2022)?